.github/workflows/build.yml

```
@@ -329,7 +329,7 @@ jobs:
           fail_ci_if_error: false
 
   minver:
-    name: Check minimum versions of direct dependencies
+    name: Test minimum versions of direct dependencies (with all features)
```

I don't think we need to adapt the name here (which is a little tricky with merge queue setup).

---

Disagreed on my part, too bad about the issue with the merge q, reverting this update anyways ...
```suggestion
    name: Check minimum versions of direct dependencies
```